asserts/database.go

```diff
@@ -99,28 +92,11 @@ func OpenDatabase(cfg *DatabaseConfig) (*Database, error) {
 	bs := cfg.Backstore
 	keypairMgr := cfg.KeypairManager
 
-	// falling back to at least one of the filesytem backstores,
-	// ensure the main directory cfg.Path
-	// TODO: decide what should be the final defaults/fallbacks
-	if bs == nil || keypairMgr == nil {
-		err := os.MkdirAll(cfg.Path, 0775)
-		if err != nil {
-			return nil, fmt.Errorf("failed to create assert database root: %v", err)
-		}
-		info, err := os.Stat(cfg.Path)
-		if err != nil {
-			return nil, fmt.Errorf("failed to create assert database root: %v", err)
-		}
-		if info.Mode().Perm()&0002 != 0 {
-			return nil, fmt.Errorf("assert database root unexpectedly world-writable: %v", cfg.Path)
-		}
-
-		if bs == nil {
-			bs = newFilesystemBackstore(cfg.Path)
-		}
-		if keypairMgr == nil {
-			keypairMgr = newFilesystemKeypairMananager(cfg.Path)
-		}
+	if bs == nil && keypairMgr == nil {
```

Comment:
@niemeyer that's what the TODO inside the if is about, if only one is nil and it's used by mistake we panic atm, I would prefer Null* impls though than checking in the methods, in a sense I expect that case to be covered by tests (and is more relevant for the store) because the client will use OpenSysDatabase which sets both anyway